module/gmp-openh264.cpp

```diff
@@ -708,10 +708,13 @@ class OpenH264VideoEncoder : public GMPVideoEncoder, public RefCounted {
     // Buffer up the data.
     uint32_t length = 0;
     std::vector<uint32_t> lengths;
+    unsigned char temporalId = 0;
 
     for (int i = 0; i < encoded->iLayerNum; ++i) {
       lengths.push_back (0);
       uint8_t* tmp = encoded->sLayerInfo[i].pBsBuf;
+      assert(encoded->sLayerInfo[i].uiSpatialId == 0);
```

Comment:
Spatial SVC will always disable?

Reply:
Correct, see in InitEncode:
https://github.com/cisco/openh264/blob/42fcf2091c5705fc372571005b9c0c2fcfadb36a/module/gmp-openh264.cpp#L409-L410